adapters/kiviads/kiviads.go

```diff
@@ -0,0 +1,154 @@
+package kiviads
```

Comment:
This is a copy of Compass and Boldwin adapter, it's also very similar to BeyondMedia. Do you want to make an alias?